src/shared/fetch-wrapper.js

```diff
@@ -0,0 +1,58 @@
+import "isomorphic-fetch";
```

Comment:
this needs to have a space between these two. 

see https://github.com/mcrowder65/eslint-config-mcrowder65/issues/13